internal/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -95,6 +97,9 @@ RUN TINIARCH=$(case $(arch) in x86_64) echo -n amd64 ;; ppc64le) echo -n ppc64el
   && curl -L -o /tini https://github.com/krallin/tini/releases/latest/download/tini-$TINIARCH \
   && chmod +x /tini
 
+[[- if .Requirements ]]
```

Comment:
Add entry in the changelog over requirements. Is not it a breaking change?

Reply:
This part is not a breaking change, but existing users will indeed need to add a requirements file to their existing projects to pull in the `operator_sdk.util` collection and `community.kubernetes` collection. Added to changelog and migration guide